modules/gitops-disabling-automatic-scraping-of-metrics-for-argo-cd-instances.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-argo-cd-instances.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-disabling-automatic-scraping-of-metrics-for-argo-cd-instances_{context}"]
+= Disabling automatic scraping of metrics for Argo CD instances
+
+By default, the {gitops-title} Operator automatically scrapes metrics for all Argo CD instances to measure the performance. Scraping metrics for multiple Argo CD instances in a cluster might result in excessive storage usage. 
+
+As a preventive measure, use the web console's YAML view and configure the `ArgoCD` custom resource (CR) to disable the automatic scraping of metrics for your Argo CD instance. As a cluster administrator, by disabling metric scraping for individual instances, you can give your users better control, flexibility, and stability to manage their defined namespaces.
+
+.Procedure
+
+. Set the `spec.monitoring.disableMetrics` field value to `true` on a given Argo CD instance:
++
+.Example `ArgoCD` CR
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+ name: example-argocd
+ namespace: default
+spec:
+ monitoring:
+   disableMetrics: true
+----
++
+[TIP]
+====
+Alternatively, use the following command to disable the automatic scraping of metrics in the {gitops-title} `argocd` CLI:
+
+.Example command
+[source,terminal]
+----
+$ oc patch argocd argocd-sample -n default --type='json' -p='[{"op": "replace", "path": "/spec/monitoring/disableMetrics", "value": true}]'
+----
+
+.Example output
+[source,terminal]
+----
+argocd.argoproj.io/example-argocd patched
+----
+====
+
+
+. Verify that the `openshift.io/cluster-monitoring` label value is `false` for the `namespace` resource, for example, the `default` namespace.
```

Comment:
If you want to add in a verification step

`$ oc get ns openshift-gitops -o jsonpath='{.metadata.labels.openshift\.io\/cluster-monitoring}'`

Reply:
@varshab1210 Thanks for the inputs. But let us focus on the UI-related instructions for verification steps.